### PR TITLE
Helm3 warning

### PR DESCRIPTION
**Pull request by yspotts**
add a section in the readme about helm2 vs helm3

I also set the default value of false. Not sure if we want to keep this... Nice for helm 3 but could mess up those using helm 2

### CODE REVIEW

File: charts/kong/values.yaml [hunk 1]
```diff
@@ -205,7 +205,7 @@ ingressController:
     # If not set and create is true, a name is generated using the fullname template
     name:
 
-  installCRDs: true
+  installCRDs: false
```
Comment (hbagdi) [inline charts/kong/values.yaml]: Please do not change the default. A significant, maybe even the majority, of users are using Helm v2.

Comment (hbagdi) [reply charts/kong/values.yaml]: Please upgrade the above description accordingly as well.

Comment (yspotts) [reply charts/kong/values.yaml]: will change back. 

sorry  - when you say "the above description" do you mean in the readme regarding the default value?

Comment (hbagdi) [reply charts/kong/values.yaml]: Yes and the text in Helm 2 vs 3, where applicable.

Comment (yspotts) [reply charts/kong/values.yaml]: got it! I will remove this commit from the PR

**Review by hbagdi — APPROVED** [review]
Looks good. Please resolve the conflict.

**Comment by lindhe** [comment]
Which conflict?

**Comment by yspotts** [comment]
Conflict resolved

**Comment by hbagdi** [comment]
@yspotts Thank you for your contribution. Please fill in the contributor form to claim your Kong swag: https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#contributor-t-shirt.

**Pull request merged**